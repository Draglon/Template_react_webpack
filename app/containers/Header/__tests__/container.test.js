// import React from 'react';
// import configureStore from 'redux-mock-store';
// import { shallow } from 'enzyme';
// import toJson from 'enzyme-to-json';

// import ConnectedHeader, { Header } from '../component';

describe('<Header /> component', () => {
  // const mockStore = configureStore();
  // const initialState = {
  //   isLogged: true,
  //   removeSessionId: () => {},
  // };
  // const props = {
  //   removeSessionId: () => {},
  // };
  // let store;
  // let component;

  // beforeEach(() => {
  //   store = mockStore(initialState);
  //   component = shallow(<ConnectedHeader store={store} {...props} />);
  // });

  it('Render property', () => {
    // expect(toJson(component)).toMatchSnapshot();
  });
});
