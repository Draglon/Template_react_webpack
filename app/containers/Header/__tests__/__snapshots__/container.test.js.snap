// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> component Render property 1`] = `
<header
  className="header"
>
  <div
    className="header__wrapper"
  >
    <nav
      className="navbar"
    >
      <Link
        className="navbar__link"
        replace={false}
        to="/"
      >
        Home
      </Link>
    </nav>
    <div
      className="header__btns"
    />
  </div>
</header>
`;
